### PR TITLE
add speaker guide

### PR DESCRIPTION
**Pull request by yochannah**
This is an orphaned page - I wasn't quite sure where to link to it?

### CODE REVIEW

**Comment by yochannah** [comment]
okay, I've linked to it from the resources section in OLS-2!

File: ols-speaker-guide.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+**Online Zoom calls:**
+
+-   These calls take place online via Zoom, a link for which will be shared with you, usually via the cohort call notes.
+
+-   You can check if Zoom works on your system by using this link: [http://zoom.us/test](http://zoom.us/test).
+
+**Call notes:**
+
+-   The OLS team should have shared a draft of cohort call agenda and details in your invitation email, which are also linked to the online schedule:
```
Comment (malvikasharan) [inline ols-speaker-guide.md:69]:
```suggestion
-   The OLS team will share a collaborative document with the cohort call agenda in your invitation email, which is also linked to the online schedule:
```

File: ols-speaker-guide.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ [https://openlifesci.org/ols-2/schedule](https://openlifesci.org/ols-2/schedule)
+
+-   These notes will be available for all our participants for future reference.
+
+-   You are welcome to contribute to the notes from your session - a link for which will be shared with you.
```
Comment (malvikasharan) [inline ols-speaker-guide.md:74]:
```suggestion
-   You are welcome to contribute to the notes from your session.
```

File: ols-speaker-guide.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+-   You can [make a copy of this presentation](https://docs.google.com/presentation/d/18MwCMBrJL7LwbFvy76il-rk_wBuXq5T2oQ9iHrsvegg/edit?usp=sharing) to start creating your presentation online. It would be great if you can share the link of your talk with the OLS team before the call.
+
+**Call recording:**
+
+-   Please note that this call will be recorded for our participants, especially those who can't attend this call in-person. These recordings will be shared on our [official Youtube channel](https://www.youtube.com/channel/UCs12-ZgnDJOWIWN3Vo1XHXA).
+ If your time allows, you can also watch a video from one of our
+ previous calls (here is the [full
+playlist](https://www.youtube.com/playlist?list=PL1CvC6Ez54KAtOhuCv8e8dqaHkUylUYDB)).
+
+-   You can turn off your webcam before joining the call if you do not want to appear on the recording and on if/when you don't mind.
```
Comment (malvikasharan) [inline ols-speaker-guide.md:117]:
```suggestion
-   You can turn off your webcam before joining the call if you do not want to appear on the recording and turn it on if/when you don't mind being on the video.
```

**Review by malvikasharan — APPROVED** [review]
Looks good. I have left a few minor suggestions. 🌺

**Comment by yochannah** [comment]
thats for the review and the suggestions! 🌺

**Pull request merged**